![Image of DaSH EU banner](https://cloud.githubusercontent.com/assets/8570385/19779491/f8e4c816-9c46-11e6-9e16-a5eea6da15a6.png)
DaSH EU 2016
=========

#History:
Three DaSH events have been held, in Bethesda, La Jolla and Minneapolis.<br />
Gottfried Fischer is hosting the fourth event in Austria.

#What:
3 primary focus areas

1. Feature Service Ecosystem: building up the tools for getting data in/out of this service in preparation for the 17th IHIW
<img src="https://cloud.githubusercontent.com/assets/8570385/19779508/07e3a882-9c47-11e6-9e7c-b90f0050a2fd.png" align="right" \> 
  - REST storage and unique ID generation for locus/term/rank/sequence
  - Annotation Service: convert consensus to locus/term/rank/sequence
  - GFE Service: use feature service to generate 17th IHIW names
  - GL Service: strict relative to particular GFE Service instance (GFE-GL)
  - Bulk loading of full-gene NGS data: NMDP 80,000 individuals, other
  - Public Curation: associate a “user” with a contribution (HML/cons.)
  - Private Curation: segregate PHI/PII from genotype data
  - Support beyond classical HLA to non-classical HLA and KIR genomics
  - Upload to (and download from) SRA

2. HL7 FHIR® (Fast Healthcare Interoperability Resources). We will be exploring the use of HL7 FHIR® to exchange HLA typing dating with the EMRs and other healthcare systems. To prepare for this, the following will be provided ahead of the meeting
  - FHIR® resource/profile documentation and introductory material
  - Examples of how NGS based genotyping data may be represented in FHIR®
  - Link to a FHIR® compliant server which we will use to exchange data
  - Webinar presenting an introduction to FHIR® and exchanging HLA genotyping data.
For some introductory information, see the Proposed Standard for Trial Use 3 (STU3) at http://hl7.org/fhir/2016Sep and the wiki documentation at http://wiki.hl7.org/index.php?title=FHIR

3. Global Haplotype Analysis. Curation/Creation/Population services. Quality metrics. Global HF analysis. AFND/Publication. Licensing and data access.

##Other topics:
- HML adoption
  - Dissemination of the tools to the typing community.
  - A publically available pipeline for the analysis of NGS data, which can be used as reference for the validation of new alleles or other typing software.
  - A tool that allows the generation of HML from current NGS SW vendor reports.
  - Long read data: PacBio, Oxford Nanopore.
  - Revisit KIR presence/absence and CNV reporting.
  - Long term support of services & interaction with societies: EFI/APHIA/ASHI.
- Non-coding variation such as the HLA-C miRNA
  - Mining and reporting known functional variants from outside codons.
  - Automated annotation of non-coding features.
# When: 
The meeting will start with an informal event on Thursday night, November 3rd. We will work from 8:00 am on Friday 4th, finishing at 4:00 pm on Saturday 5th November 2016, stopping when appropriate to eat and drink and socialize.

# Where:
On the campus of the University of Vienna, Austria.

# Further details:
Please contact Michael Wright, mwright@nmdp.org

As the name 'hackathon' implies, the meeting is geared to 'implementers,' and less so to 'talkers' and 'directors.' 
