DaSH 2016
=========

We will definitely work on <br>
(i) The Feature service that is currently available (http://feature.nmdp-bioinformatics.org/) (https://github.com/nmdp-bioinformatics/service-feature), with the aim that this service can serve as the hub for a set of services for recording and describing sequence features; some of those services and tools can be  developed/prototyped at the Hackathon.<br> 
(ii) Standardized reporting using GL string, e.g., DRB3/4/5 results.  We suggest that the GL String service should be regarded as the format for reporting HLA genotypes rather than the NMDP format. How should we expand GL string and MIRING to areas not adequately covered e.g. copy number, missing loci, loci not tested.
<br>
These other topics can be regarded as options; with it being a Hackathon the topics covered are often steered by the people in the room.<br>
(a) Persisting amino acids.<br>
(b) Converting to and from VCF and to and from PED formats.<br>
(c) Phase issues and how to represent them/collect un-phased data in consensus sequences.<br>
(d) Finishing the last mile of implementation: authentication, etc.<br>
(e) A tool for applying coordinate systems to consensus sequence, and then identifying the input parameters for the feature service. <br>
(f) Accepting/annotating sequence blocks, and annotating the potential matches<br>
(g) Predicting known splice errors/variants for HLA, and providing some annotation for that.<br>
(h) Graph based assembly (FASTG, GA4GH, etc.).<br>
(i) New features, other tools for applying additional constraints or parsing<br>

As the name 'hackathon' implies, the meeting is geared to 'implementers,' and less so to 'talkers' and 'directors.' 